src/resty/ljsonschema/init.lua

```diff
@@ -727,7 +727,10 @@ generate_validator = function(ctx, schema)
       elseif schema.format == "date-time" then
         local split_pattern = "^(.+)[tT](.+)$"
         ctx:stmt(sformat('  local date_value, time_value = %s:match(%q)', ctx:param(1), split_pattern))
-      end
+        ctx:stmt(        '  if not date_value then')
+        ctx:stmt(sformat('    return false, %s([[expected valid %q, got %%q]], %s)', ctx:libfunc('string.format'), schema.format, ctx:param(1)))
+        ctx:stmt(        '  end')
+        end
```

Comment:
nit: spacing

Reply:
not sure what you mean here?

Reply:
The last `end` is not properly indented matching up with the `elseif`

```diff
      elseif schema.format == "date-time" then
        local split_pattern = "^(.+)[tT](.+)$"
        ctx:stmt(sformat('  local date_value, time_value = %s:match(%q)', ctx:param(1), split_pattern))
-      end
+        ctx:stmt(        '  if not date_value then')
+        ctx:stmt(sformat('    return false, %s([[expected valid %q, got %%q]], %s)', ctx:libfunc('string.format'), schema.format, ctx:param(1)))
+        ctx:stmt(        '  end')
+      end
```

Reply:
ah, thx! fixed